projects/plugins/boost/tests/e2e/specs/common.test.js

```diff
@@ -0,0 +1,57 @@
+import { test, expect } from '../fixtures/base-test.js';
+import { DashboardPage, PluginsPage, Sidebar } from 'jetpack-e2e-commons/pages/wp-admin/index.js';
+import { execWpCommand } from 'jetpack-e2e-commons/helpers/utils-helper.cjs';
+import { boostPrerequisitesBuilder } from '../lib/env/prerequisites.js';
+import { prerequisitesBuilder } from 'jetpack-e2e-commons/env/prerequisites.js';
+import { JetpackBoostPage } from '../lib/pages/index.js';
+
+test.describe( 'Common', () => {
+	test.afterAll( async ( { browser } ) => {
+		const page = await browser.newPage();
+
+		await prerequisitesBuilder( page ).withActivePlugins( [ 'boost' ] ).build();
+		await boostPrerequisitesBuilder( page ).withConnection( true ).build();
+	} );
+
+	test( 'Click on the plugins page should navigate to Boost settings page', async ( { page } ) => {
+		await DashboardPage.visit( page );
+		await ( await Sidebar.init( page ) ).selectInstalledPlugins();
+		const pluginsPage = await PluginsPage.init( page );
+		const selector = "tr[data-slug='jetpack-boost'] .row-actions a[href*='=jetpack-boost']";
```

Comment:
Thank you for the explanation. It makes sense and making use of the Page Objects Model makes the tests cleaner and more readable.

I have made some changes in commit e2a724d. However, you will notice that I did not remove all of them in the [critical-css.test.js](https://github.com/Automattic/jetpack/blob/add/boost-e2e-tests/projects/plugins/boost/tests/e2e/specs/critical-css.test.js), [lazy-images.test.js](https://github.com/Automattic/jetpack/blob/add/boost-e2e-tests/projects/plugins/boost/tests/e2e/specs/lazy-images.test.js) and [render-blocking-js.test.js](https://github.com/Automattic/jetpack/blob/add/boost-e2e-tests/projects/plugins/boost/tests/e2e/specs/render-blocking-js.test.js) because the selectors being testes (such as  `#jetpack-boost-critical-css`, `.jetpack-lazy-image`, and `#blockingScript`) do not really belong to any single page really. I am not sure if it is acceptable to leave it as is, or maybe you might have a suggestion?